.babelrc

```diff
@@ -1,4 +1,4 @@
 {
-  "presets": [["env", { "modules": false }], "react"],
+  "presets": [["env", { "modules": false }], "stage-3", "react"],
```

Comment:
[stage-3](https://babeljs.io/docs/plugins/preset-stage-3/) includes rest, spread, and async generators. 

I was thinking maybe using [stage-2](https://babeljs.io/docs/plugins/preset-stage-2/) for class property transforms.

Looks like Folio's current modules use stage-0